renovate/dependencyDashboard.js

```diff
@@ -0,0 +1,30 @@
+export class Dependency {
+  constructor(name, version) {
+    this.name = name;
+    this.version = version;
+  }
+}
+
+export const getDependenciesForRepo = ({ octokit }, repo) => {
+  return octokit
+    .request(repo.issuesUrl)
+    .then((response) => {
+      const dependencies = response
+        .data
+        .find((issue) => issue.user.login === "renovate[bot]" && issue.pull_request === undefined)
+        ?.body
+        ?.split(/\r?\n|\r|\n/g)
+        ?.map((line) => {
```

Comment:
I find this function quite hard to parse. I think if I had to jump into this repo to do maintenance it would take me quite a while to work out what is going on. 
Do you think it would be clearer if it was broken down into smaller functions that can be named to demonstrate to the reader what is being done?

Reply:
Done.